src/components/ScrollableTabBar/ScrollableTabBar.js

```diff
@@ -113,7 +112,7 @@ class ScrollableTabBar extends React.PureComponent {
           }
         ]}
       >
-        <NestedScrollView
+        <ScrollView
```

Comment:
Have you checked if this solution works on Android correctly ?

Reply:
Yes, it works good on versions 0.60.0+
I think ScrollView was fixed by React Native